MobileCenterDistribute/MobileCenterDistributeTests/MSBasicMachOParserTests.m

```diff
@@ -0,0 +1,54 @@
+#import <XCTest/XCTest.h>
+#import "MSBasicMachOParser.h"
+#import "MSUtil.h"
+
+@interface MSBasicMachOParserTests : XCTestCase
+@end
+
+@implementation MSBasicMachOParserTests
```

Comment:
You can probably add a unit test to cover a read failure from `readDataFromFile`. It's a private method though so we have this convention to expose private methods to be testable: Create a new file called `MSBasicMachOParserPrivate`, there you can create an anonymous category  to extend the MSBasicMachOParser class and eventually expose the `readDataFromFile` method. Take a look at some other private files if you want.